linux-version/Cargo.toml

```diff
@@ -13,4 +13,4 @@ nix = "0.17"
 semver = "0.9"
 
 [build-dependencies]
-bindgen = { version = "0.51", default-features = false }
\ No newline at end of file
+bindgen = { version = "0.57", default-features = false }
```

Comment:
was this upgrade required?

Reply:
yes, otherwise there was a conflict on native crate `clang-sys`, linux-version transitively depends on v0.28.0, but userfaultfd-sys depends on ^1